src/assets/stylesheets/scrivitoEditing.scss

```diff
@@ -5,3 +5,8 @@
 [contenteditable]:focus-within {
   text-transform: none;
 }
+
+.img-flag[data-scrivito-image-placeholder] {
+  min-height: unset !important;
+  width: 17px !important;
```

Comment:
The placeholder is now using the correct space, but it still visible for a brief moment:
<img width="628" alt="Screenshot 2025-04-01 at 10 17 26" src="https://github.com/user-attachments/assets/6361c46f-26e0-4c92-b6dd-a438da2328f8" />

Full video: 

https://github.com/user-attachments/assets/4e18b6b0-7325-4566-9df8-b28c7c5fbca8

Can we hide the image placeholder altogether?